js/codemeta_generation.js

```diff
@@ -198,16 +227,39 @@ function importPersons(prefix, legend, docs) {
         return;
     }
 
+    const roles = {};
+    for (const doc of docs) {
+        if (doc['@type'] === 'Role') {
+            let authorId = doc['author']['@id'];
+            authorId = isBlankNodeId(authorId) ? authorId.substring(2) : authorId;
```

Comment:
As far as I can tell, there is nothing preventing a document to have both `https://example.org/` as IRI and `_:https://example.org/` as blank node identifier. Stripping `_:` would make them collide. Why do you need to strip the prefix?